Gems/Atom/Tools/AtomToolsFramework/Code/Source/AssetBrowser/AtomToolsAssetBrowserInteractions.cpp

```diff
@@ -105,7 +105,8 @@ namespace AtomToolsFramework
                     caller, QObject::tr("Run Script"), QString(AZ::Utils::GetProjectPath().c_str()), QString("*.py"));
                 if (!script.isEmpty())
                 {
-                    AZStd::vector<AZStd::string_view> pythonArgs{ entry->GetFullPath() };
+                    AZStd::string shaderPath = entry->GetFullPath();
+                    AZStd::vector<AZStd::string_view> pythonArgs{ shaderPath };
```

Comment:
These lines can be reverted. This class should not be referring to anything shader specific. I'm making the proposed changes to this file and the main menu to support registering script paths with the settings registry. That should be up for PR shortly.